Makefile

```diff
@@ -24,6 +24,7 @@ backend-typing:
 install-python-dependencies:
 	pip install -e .
 	pip install -r requirements-test.txt
+	pip install git+https://github.com/getsentry/arroyo@d4caa8d3b267c30efb5a996545cdd0e5ce5cb7b2#egg=sentry-arroyo
```

Comment:
This needs undoing before merge with a bump to a new arroyo release.

Reply:
This should unblock:
https://github.com/getsentry/snuba/pull/2670